docs/body.md

```diff
@@ -390,32 +407,71 @@ Instances
 
 The following device level metrics are collected:
 
-* Flavors
-* Total (count) Images
+#### Flavors
+
 * Total (count)
-* Total count per image state Servers
+
+#### Images
+
+* Total (count)
+* Total count per image state
+
+#### Servers
+
 * Total (count)
 * Total count per server state
-* Queues
+
+#### Queues
+
 * Event (count)
-* Performance (count) Agents
+* Performance (count)
+
+#### Agents
+
 * Total (count)
-* Total count per agent type Networks
+* Total count per agent type
+
+#### Networks
+
 * Total (count)
-* Total count per network state Routers
+* Total count per network state
+
+#### Routers
+
 * Total (count)
-* Total count per router state Volumes
+* Total count per router state
+
+#### Volumes
+
 * Total (count)
-* Total count per volume state Volume Snapshots
+* Total count per volume state
+
+#### Volume Snapshots
+
 * Total (count)
-* Total count per volume snapshot state Volume Pool
+* Total count per volume snapshot state
+
+#### Volume Pool
+
 * Total (count)
 * Total count per volume pool state
 
-Note: All events processed through Ceilometer are automatically 
-exposed via the Zenoss Event Console, and all metrics collected by Ceilometer
-may be collected and graphed in Zenoss through the use of custom monitoring 
-templates.
+#### Notes:
+
+* All events processed through Ceilometer are automatically 
+  exposed via the Zenoss Event Console, and all metrics collected by Ceilometer
+  may be collected and graphed in Zenoss through the use of custom monitoring 
+  templates.
+
+* The Instance metrics for *Disk IO Rate* were removed in OpenStack version
+  Queens and later. In order to continue to see those you need to edit the 
+  OpenStack Ceilometer configuration /etc/ceilometer/polling.yaml and add the
+  following to the *meters* section:
+
+        - disk.read.bytes
+        - disk.read.requests
+        - disk.write.bytes
+        - disk.write.requests
```

Comment:
I don't necessarily want to suggest that people should do this unless they need it.   I would probably say that they have been deprecated, that they can be re-enabled as you describe, but that they may be removed completely in a future version of openstack.